src/v/kafka/server/tests/alter_config_test.cc

```diff
@@ -343,7 +343,8 @@ FIXTURE_TEST(
       "message.timestamp.type",
       "redpanda.datapolicy",
       "redpanda.remote.read",
-      "redpanda.remote.write"};
+      "redpanda.remote.write",
+      "max.message.bytes"};
```

Comment:
nit: this seems unrelated to describe_configs but ended up in the same commit

Reply:
this test `test_topic_describe_configs_requested_properties` tests describe config

Reply:
ah, makes sense!